controllers/chat/group/groupAddMembers.js

```diff
@@ -72,6 +72,8 @@ const groupAddMembers = async (req, res) => {
         text: textDefaultUser,
         own_text: textOwnUser,
         other_text: textTargetUser,
+        other_system_user: member,
+        better_type: 'add_member_to_group',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `other_system_user` and `better_type` properties to the object seems fine. However, it's important to ensure that these new properties are handled correctly wherever this object is used. If not, it could lead to unexpected behavior or crashes. Please make sure that all functions or methods that use this object can handle these new properties.

```diff
+        other_system_user: member,
+        better_type: 'add_member_to_group',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->